tests/Prometheus/dashboard_static.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/sh
 
 make DATASOURCE=Prometheus \
-     JOB=tarantool_app \
-     RATE_TIME_RANGE=2m \
+     JOB=tarantool \
```

Comment:
I didn't got why `RATE_TIME_RANGE=2m` is deleted here, but still exist in dashboard_tdg_static.sh?

Reply:
It was a mistake, fixed. Thank you!